process_manifest/iiifManifest.py

```diff
@@ -116,14 +129,19 @@ def _convert_label_value(self, label, value):
             }
         return None
 
+    def _lang_wrapper(self, line):
+        return {self.lang: [line]}
+
     def _add_provider(self, provider):
-        if (provider == 'embark'):
+        if (provider.lower() == 'embark'):
             return self._snite_proivider()
-        elif (provider == 'archivespace'):
+        elif (provider.lower() == 'archivesspace'):
             return self._archives_proivider()
-        elif (provider == 'rbsc'):
+        elif (provider.lower() == 'rbsc'):
             return self._rbsc_proivider()
 
```

Comment:
Do we want to go ahead and add an 'aleph' provider, since I hope to be adding aleph processing soon